rtlib/include/rt/ImageStitcher.hpp

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#include <vector>
+#include <iostream>
+
+#include <opencv2/stitching.hpp>
```

Comment:
```suggestion
#include <opencv2/core.hpp>
#include <opencv2/stitching.hpp>
```
Also, see if you can remove the header dependency on `stitching` by simplifying what private variables you're keeping around.